test/integration_test/common_test.go

```diff
@@ -159,6 +159,21 @@ func setup() error {
 	return nil
 }
 
+func constructDefaultStorageCluster() (*corev1.StorageCluster, error) {
```

Comment:
this might be a little confusing, it's more like an empty stc, we populate the default values upon stc creation, see `createStorageCluster`

Reply:
createStorageCluster as it is in controller_test.go which is not in the test util to be reused by other tests. slightly different, the function here is to take the manifect URL to generate the storage cluster as it contains the images.

Reply:
I think he means this one - https://github.com/libopenstorage/operator/blob/master/test/integration_test/common_test.go#L190

Reply:
that's different API, that API creates stc to kuberntes, while this construct*** API just create in memory object.

Reply:
I mean the function name is confusing as the stc created only has basic images in it but not having other default specs populated. we can have a shared utility function like this but better to merge this function with `constructStorageCluster` directly, they serve the same purpose.

Reply:
i will rename it

Reply:
actually i would set portworx defaults and keep the name in this API so it can be reused